_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -923,6 +947,87 @@ In this strategy, it is expected to have IITB and CFOB work together for during
 | TSDM    | Target Solution Delivery Model            |
 | TSWG    | Technical Standards Working Group         |
 
+## Appendix F - Statistics regarding Large IT-Enabled Projects
+
+The following are statistics and references regarding the problems and success rates of large IT-enabled Projects.
+
+**1) Standish Group study**
+
+The Standish Group, a research advisory organization that focuses on software development performance[^16], found that “of 3,555 projects from 2003 to 2012 that had labor costs of at least $10 million, only 6.4% were successful. The Standish data showed that 52% of the large projects were “challenged” meaning they were over budget, behind schedule or didn't meet user expectations. The remaining 41.4% were failures — they were either abandoned or started anew from scratch."[^17]
+
+The standish group study and results were also mentioned in Chapter 3 of the Novembre 2006 Report of the Auditor General of Canada (statements 3.5 and 3.6). See point 3 below
+
+**2) 2016 and 2019 House of Commons Questions (projects of more than $1M)**
+
+The Canadian Digital Service (CDS) [released a dataset](https://large-government-of-canada-it-projects.github.io/)[^18] collected in response to two written questions in the House of Commons, from June 2016 and May 2019. Each question asked federal government departments to report ongoing or planned IT projects over $1M.
```

Comment:
It wasn't CDS...It was Sean and Lucas but the were doing that work as part of a CivicTech project (so not part of their day jobs).  Would rephrase to "Thanks to an Ottawa Civic Tech project, a dataset on large government IT projects was released with responses collected from two written...."